analyses/molecular-subtyping-MB/03-compare-classes.Rmd

```diff
@@ -33,15 +33,28 @@ suppressPackageStartupMessages(library(plyr))
 root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
 data_dir <- file.path(root_dir, "data")
 
+# This was created in 00-mb-select-pathology-dex.Rmd
+clinical_file <- file.path(data_dir, "pbta-histologies.tsv")
+
 # set results directory
 results_dir <- file.path(root_dir, "analyses", "molecular-subtyping-MB", "results")
+
+# this file was created in the 01-filter-and-batch-correction.R
+subset_clin_file <- file.path("input", "subset-mb-clinical.tsv")
+```
+
+```{r}
+# Read in the JSON file that contains the strings we'll use to include or
+# exclude samples for subtyping - see 00-HGG-select-pathology-dx
+path_dx_list <- jsonlite::fromJSON(terms.file)
 ```
```

Comment:
I think this can go now with your most recent update.

Reply:
I just found that. It's deleted in the next commit.